### PR TITLE
added pyscss 1.3.5

### PR DESCRIPTION
**Pull request by stuertz**

### CODE REVIEW

File: recipes/pyscss/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: True
```
Comment (jakirkham) [inline recipes/pyscss/meta.yaml]: Why keep the egg directory?

File: recipes/pyscss/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - pyscss=scss.tool:main
+    - less2scss=scss.less2scss:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - enum34
+    - pathlib
```
Comment (jakirkham) [inline recipes/pyscss/meta.yaml]: Please drop anything from `build` that is not strictly needed for building. Probably should just be `python` and `setuptools`.

File: recipes/pyscss/meta.yaml (hunk 1 above)
Comment (stuertz) [inline recipes/pyscss/meta.yaml]: Not needed.

File: recipes/pyscss/meta.yaml (hunk 2 above)
Comment (stuertz) [inline recipes/pyscss/meta.yaml]: done.

File: recipes/pyscss/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    - scss.extension.compass
+    - scss.grammar
+
+  commands:
+    - pyscss --help
+    - less2scss --help
+
+about:
+  home: http://github.com/Kronuz/pyScss
+  license: MIT
```
Comment (jakirkham) [inline recipes/pyscss/meta.yaml]: Please add `license_file: LICENSE`.

Comment (stuertz) [inline recipes/pyscss/meta.yaml]: done.

**Pull request merged**